java/vector/src/main/java/org/apache/arrow/vector/types/pojo/DictionaryEncoding.java

```diff
@@ -65,6 +81,11 @@ public Int getIndexType() {
     return indexType;
   }
 
+  @JsonGetter("isDelta")
+  public boolean isDelta() {
```

Comment:
So this means: if true, the associated Dictionary is a delta dictionary for a previous Dictionary with the same ID.